blogs/qserial.md

```diff
@@ -0,0 +1,243 @@
+Hey AngularJS devs!  Let's talk about executing async tasks serially with promises.
+
+First, we notice .then allows our success/fail callbacks to return a promise.  $q treats success/fail callbacks that return promises specially.  We'll look at two examples to see how $q treats success/fail functions that return promises differently from those that don't.
+<!--more-->
+
+Normal success function:
+<pre>
+$q.when()  
```

Comment:
Yes, but much more.  It turns any object into a promise.

So 

```
var promise = $q.when(undefined);
var promse = $q.when({ foo: bar });
var promise = $q.when(previousPromise);
```

It's a really handy function for dealing with objects that may not be promises.  This is often the case when you're accepting parameters in a function or dealing with third party apis.